### DISCUSSION TITLE
Support for multiple databases

### DISCUSSION THREAD

**Discussion by hesperanca** [IEvangelist/azure-cosmos-dotnet-repository#209] [Q&A] [answered]
Hi,

Is it possible to use more than one database? I've noticed that we can set ContainerPerItemType to true which allow us to have further control about the containers we use but unfortunately we can't seem to be able to specify which database each container will reside. It looks like it's assumed that all containers will be in the same database.

Just to give a bit of context we are building a multitenant app where configuration data (e.g. tenant information) is to be kept in a separate DB (Admin DB) from the tenant data. Also, further down the line, there is a requirement where some tenants (on a more expensive plan) will be given the option to have their own DB. 

Many thanks in advance.

Kind Regards,
Hugo

**Comment by IEvangelist** [comment, marked as answer]
More and more users are asking this, it might be something that we have to start taking more seriously. I love the idea, it's just a lot to change, test, and implement. Let's continue the discussion here: https://github.com/IEvangelist/azure-cosmos-dotnet-repository/discussions/271
